### ISSUE TITLE
add support for more pages

### ISSUE THREAD

**Issue by ekm507** [ekm507/mozilfun#7]
Hi
pages to support in the future:
- categories (e.g. [this](https://addons.mozilla.org/en-US/firefox/extensions/category/appearance/))
- developer profile (e.g. [this](https://addons.mozilla.org/en-US/firefox/user/14103783/))
- add-on versions (e.g. [this](https://addons.mozilla.org/en-US/firefox/addon/turbo-download-manager/versions/?utm_content=featured&utm_medium=referral&utm_source=addons.mozilla.org))

regards.

**Comment by nimafanniasl** [comment]
Yeah, we have some links like (---developer-profile---) that don't work.
